config/providers.yml

```diff
@@ -17,5 +17,12 @@
       {display_name: us-east-a2, internal_name: leaseweb-wdc02, ui_name: "Virginia, US", visible: true},
       {display_name: us-south-a13, internal_name: leaseweb-dal13,  ui_name: "Dallas, US", visible: false},
     ]
+  },
+  {
+    provider_display_name: Latitude,
+    provider_internal_name: latitude,
+    locations: [
+      {display_name: eu-central-l1, internal_name: latitude-fra, ui_name: "Frankfurt, Germany", visible: false}
```

Comment:
We use `a` such as `us-east-a2` for Leaseweb. Do you know the reason for that (I don't know)? Should we use `a` for Latitude too? Lower `l` is little hard to differentiate from numbers such as `1`?

Reply:
this makes sense. I'll update